easy_transformer/EasyTransformer.py

```diff
@@ -82,6 +82,8 @@ def __init__(
                 self.tokenizer.pad_token = self.tokenizer.eos_token
             if self.tokenizer.bos_token is None:
                 self.tokenizer.bos_token = self.tokenizer.eos_token
+            if isinstance(self.tokenizer, PreTrainedTokenizer):
```

Comment:
I imagine the error arises since the tokenizer is not a PreTrainedTokenizerFast, which has different syntax